tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,11 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(rd2html("\\link[base]{library}"),
+               link_remote(label="library", topic="library", package="base"))
```

Comment:
> The test would be more informative if the first and second arguments weren't the same value.

Sorry if I'm being obtuse, but I'm not quite getting this point. Do you mean that you would have liked a test that included a static string as baseline (as I explained this would be redundant with [this link_remote test](https://github.com/jefferis/pkgdown/blob/master/tests/testthat/test-link.R#L21-L24)). Or do you mean you would have liked a more complex expression as input in combination with an `expect_match` expectation? Or something else?

> Also please use spaces around =, and if you're going break a function across multiple lines, indent them both by two spaces.

Like so?

```r
test_that("can convert cross links to online documentation url", {
  expect_equal(
    rd2html("\\link[base]{library}"),
    link_remote(label = "library", topic = "library", package = "base")
  )
})
```